### ISSUE TITLE
spotify.link links aren't detected

### ISSUE THREAD

**Issue by laralove143** [vleeuwenmenno/songwhip-bot#16]
People can use links like `https://spotify.link/{song}` to link to a song, Songwhip converts those properly, so it's only a matter of the bot detecting them

**Comment by vleeuwenmenno** [comment]
This is a new link domain, I think? I'll have a look and add it to the filter tomorrow evening to see if I can add this.

**Comment by Podrock** [comment]
Yea this is something new Spotify is doing apparently.

**Comment by Podrock** [comment]
For what it's worth I've discovered this is the default share method from the Spotify mobile app. The desktop app still does open.spotify.com

**Comment by laralove143** [comment]
they'll probably roll it over to desktop soon, so we should fix this, it shouldnt be that much work so i could create a pr for this

**Comment by vleeuwenmenno** [comment]
Sadly, my whole PC died yesterday and I only just got it back up and running with a clean installation. It's 4:25 at night/morning, so I'm going to catch some sleep, but feel free to make a PR. Otherwise, I'll look into this first thing in the afternoon.

**Comment by laralove143** [comment]
oh no pressure! if you wanna do it afternoon is more than enough

**Comment by vleeuwenmenno** [comment]
As of v1.5.4 this is solved temporarily. I am planning a more permanent fix, explained here https://github.com/vleeuwenmenno/songwhip-bot/issues/17#issuecomment-1732627728

Tldr: server level config allowing for updating these links/domains without having to release a new bot build version.
